### PR TITLE
starter: redesign starter

### PR DESCRIPTION
**Pull request by zanettin**
# What is it?

- [x] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description
- [x] Reworked the starter app to match CI
- [x] moved `starter` specific routes, styles and components into a dedicated directory to make it easier to remove them
- [x] added new gauge component 
- [x] rework `flower` app
- [x] rework `todo` app

## Video

https://user-images.githubusercontent.com/3241476/232895102-6f081c42-9b58-4060-ba7b-a37f6751f8d4.mp4

## Design
localhost_5173_

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by zanettin** [comment]
there would be also an option to get a perfect lighthouse score when changing the front color of the blue button and on the light purple sections. otherwise we would go with **100/96/100/100**
localhost_5173_

File: starters/apps/basic/src/components/starter/gauge/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { component$ } from '@builder.io/qwik';
+import styles from './gauge.module.css';
+
+export default component$(({ value = 50 }: { value?: number }) => {
+  const safeValue = value < 0 || value > 100 ? 50 : value;
```
Comment (manucorporat) [inline starters/apps/basic/src/components/starter/gauge/index.tsx:5]: let's inline this in the template, better performance for now

File: starters/apps/basic/src/components/starter/header/header.tsx [hunk 2]
```diff
@@ -5,28 +5,30 @@ import styles from './header.module.css';
...
-      <ul>
-        <li>
-          <a href="https://qwik.builder.io/docs/components/overview/" target="_blank">
-            Docs
-          </a>
-        </li>
-        <li>
-          <a href="https://qwik.builder.io/examples/introduction/hello-world/" target="_blank">
-            Examples
+      <div class={`container ${styles.wrapper}`}>
```
Comment (manucorporat) [inline starters/apps/basic/src/components/starter/header/header.tsx:8]: use array here

File: starters/apps/basic/src/components/starter/gauge/index.tsx (hunk 1 above)
Comment (zanettin) [reply starters/apps/basic/src/components/starter/gauge/index.tsx:5]: i need the value as well on L25 (to calc the stroke array). still better to have the evaluation twice?

File: starters/apps/basic/src/components/starter/header/header.tsx (hunk 2 above)
Comment (zanettin) [reply starters/apps/basic/src/components/starter/header/header.tsx]: oh thx 🙏 updated all cases i've found (repl) 

and added a note on the styles docs:
Bildschirmfoto_2023-04-20_um_21_27_59


let me know if the wording needs an adjustment or if you have an addition

Comment (manucorporat) [reply starters/apps/basic/src/components/starter/header/header.tsx]: I would have a single code snippets, not two

Comment (zanettin) [reply starters/apps/basic/src/components/starter/header/header.tsx]: Bildschirmfoto 2023-04-21 um 13 04 58

updated.

**Comment by manucorporat** [comment]
Rocking it man!

**Pull request merged**